src/main/kotlin/no/nav/syfo/varsel/VarselService.kt

```diff
@@ -18,14 +21,19 @@ class VarselService(
     private val senOppfolgingVarselKafkaProducer: SenOppfolgingVarselKafkaProducer,
     private val pdfgenService: PdfgenService,
     private val dokarkivClient: DokarkivClient,
+    private val behandlendeEnhetClient: BehandlendeEnhetClient,
```

Comment:
Denne kjører til at detekt klager på at det er for mange parametre i konstruktøren: LongParameterList. 
Ser at vi allerede supresser det samme i to andre steder: `SenOppfolgingControllerV1` og `SenOppfolgingControllerV1`.
Tror det beste blir å deaktivere regel i `detekt-config.yml`, foreslår å legge til denne:
```
complexity:
  LongParameterList:
    constructorThreshold:
      active: false
```

Reply:
... og fjerne` @Suppress("LongParameterList")` der det finnes

Reply:
Looks like, we can't set only `constructorThreshold` to `active: false`
I got the following error message 
```Run failed with 1 invalid config property.
  	- Unexpected nested config for 'complexity>LongParameterList>constructorThreshold'.```
We either can set the `constructorThreshold: 10` f.ex. or set ```
LongParameterList: 
active:false```
What do you prefer @Vitenok ?

Reply:
As we discussed, maybe it is ok to deactivate both 👍🏻